__device-tests__/gutenberg-editor-image.test.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @format
+ * */
+
+/**
+ * Internal dependencies
+ */
+import EditorPage from './pages/editor-page';
+import {
+	setupDriver,
+	isLocalEnvironment,
+	stopDriver,
+	isAndroid,
+	clickMiddleOfElement,
+} from './helpers/utils';
+import testData from './helpers/test-data';
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 240000;
+
+describe( 'Gutenberg Editor Image Block tests', () => {
+	let driver;
+	let editorPage;
+	let allPassed = true;
+
+	// Use reporter for setting status for saucelabs Job
+	if ( ! isLocalEnvironment() ) {
+		const reporter = {
+			specDone: async ( result ) => {
+				allPassed = allPassed && result.status !== 'failed';
+			},
+		};
+
+		jasmine.getEnv().addReporter( reporter );
+	}
+
+	beforeAll( async () => {
+		driver = await setupDriver();
+		editorPage = new EditorPage( driver );
+	} );
+
+	it( 'should be able to see visual editor', async () => {
+		await expect( editorPage.getBlockList() ).resolves.toBe( true );
+	} );
+
+	it( 'should be able to add an image block', async () => {
+		await editorPage.addNewImageBlock();
+		let imageBlock = await editorPage.getImageBlockAtPosition( 1 );
+
+		if ( ! isAndroid() ) {
```

Comment:
On the emulator, clicking add image doesn't add an image to the image block, so that's why when running the test on iOS (Add Image > WordPress media library) it actually adds an image but on Android it doesn't. 🤔  I could probably combine this combine this code into a function to make it a little less obvious, WDYT?

Reply:
Thanks for explaining! That makes sense. 

I do think it would be helpful to add a bit more context here and on line 70. I don't have any strong preference between just adding a comment versus moving this code into a function, or even just creating a local "alias" along the lines of `isCapableOfAddingImageToImageBlock() = () => ! isAndroid();`, which would then be pretty descriptive if we used it in place of the `! isAndroid()` checks.

Reply:
I just opted for more context in the comments